app/containers/CustomHeader/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { ReactNode } from 'react';
+import { PixelRatio, Text, View } from 'react-native';
+import { NativeStackHeaderProps } from '@react-navigation/native-stack';
+import { HeaderBackButton } from '@react-navigation/elements';
+
+import HeaderContainer from './components/HeaderContainer';
+import { isAndroid } from '../../lib/methods/helpers';
+import { useAppSelector } from '../../lib/hooks';
+import { useTheme } from '../../theme';
+import { styles } from './styles';
+
+interface IHeaderTitle {
+	headerTitle?: string | ((props: { children: string; tintColor?: string }) => ReactNode);
+}
+
+const HeaderTitle = ({ headerTitle }: IHeaderTitle) => {
+	const { colors } = useTheme();
+	if (!headerTitle) return null;
```

Comment:
Can you always do like this to make it easier to read?

```suggestion
	if (!headerTitle) {
	  return null;
	}
```